source/standards/running-services-as-containers.html.md.erb

```diff
@@ -0,0 +1,51 @@
+---
+title: Running services as containers
+expires: 2018-05-02
+---
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+## Principles
+
+This guidance should not be applied to development environments using containers.
+
+Where possible, web applications should use the PaaS as per the [hosting guidance](standards/hosting.html).
+
+It is not recommended to run bare containers on the PaaS, as provided by the [documentation](https://docs.cloud.service.gov.uk/#deploy-a-docker-image-experimental).
+
+If it is not possible to use the PaaS, or the service is not appropriate (such as when an amount of state is required), then guidance should be followed to ensure best practice when running your service as a container.
```

Comment:
Slightly worried that this could read by a naive user (e.g. me) as "using containers on PaaS is a way of keeping state" - which isn't really true. You shouldn't be keeping state in containers. Could we make that a little bit clearer? Ta!

Reply:
Yes, that doesn't read very well, I meant as you described, ie:

"Running a service that requires state is not suitable on the PaaS" 

I'll see if I can reword to make clearer.

Reply:
Are containers the right solution to host somewhere else if state is required? I don't think they are. It might be better to omit the part about state altogether.